src/sass/components/_header.scss

```diff
@@ -0,0 +1,38 @@
+.page-header {
+    padding-top: 24px;
+    padding-bottom: 24px;
+
+    border-bottom: 1px solid rgba(0, 0, 0, 0.2);
+
+    @include tablet-up {
+        padding-top: 32px;
+        padding-bottom: 28px;  
+    }
+
+    @include desktop-up {
+        padding-top: 28px;
+        padding-bottom: 28px;
+    }
+}
+
+.page-header__wrapper {
```

Comment:
@media screen and (min-width: 768px) and (max-width: 1279px)
.page-header__wrapper {
   align-items: flex-start;
   gap: 48px;
   display: flex;
}

this fixes the position of the menu on the tablet version. I was not able to figure out why it jinx up everything on  desktop version.